beacon_node/execution_layer/src/lib.rs

```diff
@@ -918,49 +907,48 @@ impl ExecutionLayer {
             terminal_block_number: 0,
         };
 
-        let broadcast_results = self
+        let broadcast_result = self
             .engines()
             .broadcast(|engine| engine.api.exchange_transition_configuration_v1(local))
             .await;
 
         let mut errors = vec![];
-        for (i, result) in broadcast_results.into_iter().enumerate() {
-            match result {
-                Ok(remote) => {
-                    if local.terminal_total_difficulty != remote.terminal_total_difficulty
-                        || local.terminal_block_hash != remote.terminal_block_hash
-                    {
-                        error!(
-                            self.log(),
-                            "Execution client config mismatch";
-                            "msg" => "ensure lighthouse and the execution client are up-to-date and \
-                                      configured consistently",
-                            "execution_endpoint" => i,
-                            "remote" => ?remote,
-                            "local" => ?local,
-                        );
-                        errors.push(EngineError::Api {
-                            id: i.to_string(),
-                            error: ApiError::TransitionConfigurationMismatch,
-                        });
-                    } else {
-                        debug!(
-                            self.log(),
-                            "Execution client config is OK";
-                            "execution_endpoint" => i
-                        );
-                    }
-                }
-                Err(e) => {
+        let i = 0usize;
```

Comment:
This looks horrible, but I don't want to touch the `EngineError::Api` (which contains the id) to keep the changes minimal